catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/MysqlTablePropertiesMetadata.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.mysql;
+
+import static com.datastrato.gravitino.catalog.PropertyEntry.integerOptionalPropertyEntry;
+import static com.datastrato.gravitino.catalog.PropertyEntry.stringOptionalPropertyEntry;
+import static com.datastrato.gravitino.catalog.PropertyEntry.stringReservedPropertyEntry;
+
+import com.datastrato.gravitino.catalog.PropertyEntry;
+import com.datastrato.gravitino.catalog.jdbc.JdbcTablePropertiesMetadata;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MysqlTablePropertiesMetadata extends JdbcTablePropertiesMetadata {
+  public static final String ENGINE = "ENGINE";
+  public static final String DEFAULT_CHARSET = "DEFAULT CHARSET";
+  public static final String COLLATE = "COLLATE";
+  public static final String AUTO_INCREMENT = "AUTO_INCREMENT";
+  private static final Map<String, PropertyEntry<?>> PROPERTIES_METADATA;
+
+  static {
+    PROPERTIES_METADATA =
+        Collections.unmodifiableMap(
+            new HashMap<String, PropertyEntry<?>>() {
+              {
+                put(COMMENT, stringReservedPropertyEntry(COMMENT, "The table comment", true));
+                put(
+                    ENGINE,
+                    stringOptionalPropertyEntry(ENGINE, "The table engine", false, null, false));
+                put(
+                    DEFAULT_CHARSET,
+                    stringOptionalPropertyEntry(
```

Comment:
ditto

Reply:
If the value of them has little to do with version changes, I suggest using enumeration as we can verify its validity.